src/posthog-surveys.ts

```diff
@@ -29,6 +29,9 @@ export class PostHogSurveys {
 
     getActiveMatchingSurveys(callback: SurveyCallback, forceReload = false) {
         this.getSurveys((surveys) => {
+            if (!surveys) {
```

Comment:
Since our end users using the API would face this annoying issue as well, I'd say we should always return an array from `getSurveys`, and add a test for this case :)